projects/metis/src/app/dataset/history/history.component.ts

```diff
@@ -74,13 +71,6 @@ export class HistoryComponent extends SubscriptionManager {
     }
   }
 
-  /** resetTemplateRowIndex
-  /* Resets templateRowIndex to 0
-  */
-  resetTemplateRowIndex(): void {
-    this.templateRowIndex = 0;
-  }
-
   /** rowIsStripe
   /* Increments templateRowIndex variable and returns its modulus of 2
   /* @return boolean
```

Comment:
The function `rowIsStripe()` is no longer required, and should be removed, I think? The same for `this.templateRowIndex`.